ngraph/frontend/onnx_import/src/op/resize.cpp

```diff
@@ -28,6 +29,124 @@ namespace ngraph
         {
             namespace
             {
+                static const std::unordered_set<std::string> supported_modes = {
```

Comment:
@tomdol  Could you have a look?

Reply:
It looks ok, the values match the spec of this op https://github.com/onnx/onnx/blob/master/docs/Operators.md#Resize